src/vm/eventpipejsonfile.cpp

```diff
@@ -0,0 +1,131 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include "common.h"
+#include "eventpipejsonfile.h"
+
+EventPipeJsonFile::EventPipeJsonFile(SString &outFilePath)
+{
+    CONTRACTL
+    {
+        THROWS;
+        GC_TRIGGERS;
+        MODE_ANY;
+    }
+    CONTRACTL_END;
+
+    m_pFileStream = new CFileStream();
+    if(FAILED(m_pFileStream->OpenForWrite(outFilePath)))
+    {
+        delete(m_pFileStream);
+        m_pFileStream = NULL;
+        return;
+    }
+
+    QueryPerformanceCounter(&m_fileOpenTimeStamp);
+
+    SString fileStart(W("{\n\"StackSource\" : {\n\"Samples\" : [\n"));
```

Comment:
I realize this is all throw away, but it would be good to have the trace start time (as a DateTime) in the output as well as things like the process ID process name, machine name.    You don't have to do it in this code (since it is throw away), but ultimately we want such things.